docs/2.guide/3.going-further/7.layers.md

```diff
@@ -104,6 +104,11 @@ If you want to extend a private remote source, you need to add the environment v
 If you want to extend a remote source from a self-hosted GitHub or GitLab instance, you need to supply its URL with the `GIGET_GITHUB_URL=<url>` or `GIGET_GITLAB_URL=<url>` environment variable - or directly configure it with [the `auth` option](https://github.com/unjs/c12#extending-config-layer-from-remote-sources) in your `nuxt.config`.
 ::
 
+::warning
+Nuxt Layers installed with Giget should only require Nuxt modules. Other dependencies cannot be resolved because they are installed in a nested structure (node_modules/.c12/layer_name/node_modules) rather than at the root level.
+To share dependencies between your layer and the Nuxt app, install your Nuxt Layer using a package manager instead of Giget.
```

Comment:
```suggestion
Bear in mind that if you are extending a remote source as a layer, you will not be able to access its dependencies outside of Nuxt. For example, if the remote layer depends on an eslint plugin, this will not be usable in your eslint config. That is because these dependencies will be located in a special location (`node_modules/.c12/layer_name/node_modules/`) that is not accessible to your package manager.
```